src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -101,35 +88,29 @@ const ShowPracticeQuestions = () => {
       data-testid='show-practice-questions-body'
       data-analytics-region='PQ popup'
     >
-      {section ? <SectionTitle dangerouslySetInnerHTML={{ __html: section.title }} /> : null}
-      {questionsCount === 0 && nextSection
-        ? <EmptyScreen nextSection={nextSection} />
-        : (
-          <QuestionsWrapper>
-            <QuestionsHeader>
-              <FormattedMessage id='i18n:practice-questions:popup:questions'>
-                {(msg: string) => msg}
-              </FormattedMessage>
-            </QuestionsHeader>
-            <ProgressBar total={questionsCount} activeIndex={currentQuestionIndex} />
-            {
-              selectedSectionHasPracticeQuestions && currentQuestionIndex === null
-                ? <IntroScreen />
-                : null
-            }
-          </QuestionsWrapper>
-        )
-      }
-      {
-        section && linkToTheSection
-          ? <StyledContentLink href={linkToTheSection} target='_blank' data-analytics-label='Go to link' >
-            <FormattedMessage id='i18n:practice-questions:popup:read'>
-              {(msg: string) => msg}
-            </FormattedMessage>
-            <span dangerouslySetInnerHTML={{ __html: section.title }} />
-          </StyledContentLink>
-          : null
-      }
+      <Filters />
+      <ShowPracitceQuestionsContent>
+        {section ? <SectionTitle dangerouslySetInnerHTML={{ __html: section.title }} /> : null}
```

Comment:
can we remove this condition now?

Reply:
`pqSelectors.selectedSection` returns either `null` or `LinkedArchiveTreeSection` because `selectedSection` is initially set to `null` so we could only get rid of it by creating a separate component and returning `null` internally

Reply:
could we assert that it isn't null? the expectation is that it will always be set before the modal opens right?

Reply:
I think that handling `null` is safer than asserting - especially since we want to handle opening modals after entering a URL with `modal` param
An alternative that I can see would be to create a component for `ShowPracitceQuestionsContent` and pass there `section` as prop and return `null` internally if it is falsy